src/pages/workspaces/workspace/Data.js

```diff
@@ -642,7 +645,7 @@ const WorkspaceData = _.flow(
 
   return div({ style: styles.tableContainer }, [
     !entityMetadata ? spinnerOverlay : h(Fragment, [
-      div({ style: { ...styles.sidebarContainer, width: sidebarWidth } }, [
+      (isGoogleWorkspace || isFeaturePreviewEnabled('workspace-data-service')) && div({ style: { ...styles.sidebarContainer, width: sidebarWidth } }, [
```

Comment:
Is `isFeaturePreviewEnabled('workspace-data-service'))` specific to Azure-only workspaces?

Reply:
@aaronkanzer the feature flag is not specific to Azure, but I hope we will remove the flag entirely, well before MVP. What is your concern about the flag here?

Reply:
Discussed offline in meetings - resolving!